### ISSUE TITLE
PlistBuddy getting wrong CFBundleIdentifier variable with Xcode 12.0

### ISSUE THREAD

**Issue by oeduardoal** [facebook/react-native#29545]
/usr/libexec/PlistBuddy can not found a CFBundleIdentifier variable on Xcode 12 in iPhone 11 (iOS 14.0)

## Description
I tried to run my `paz.church` application with the` yarn ios` that opens the iPhone 11 (iOS 14.0) XCode 12.0 simulator, but I get an error that cannot find the application in the build folder

## React Native version:
```
System:
    OS: macOS 11.0
    CPU: (4) x64 Intel(R) Core(TM) i5-6360U CPU @ 2.00GHz
    Memory: 21.41 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 10.16.3 - ~/.nvm/versions/node/v10.16.3/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v10.16.3/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.9.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.0, DriverKit 20.0, macOS 11.0, tvOS 14.0, watchOS 7.0
    Android SDK:
      API Levels: 27, 28, 29
      Build Tools: 28.0.3, 29.0.2, 30.0.1
      System Images: android-29 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 4.0 4.0
    Xcode: 12.0/12A8169g - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_231 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0
    react-native: 0.62.2 => 0.62.2
  npmGlobalPackages:
    *react-native*: Not Found
```

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.
1. `yarn ios` 

```
PRODUCT_BUNDLE_IDENTIFIER = paz.church;
PRODUCT_NAME = Paz.Church;
```

## Expected Results
Run on the ios 14.0 simulator (iPhone 11)

## Snack, code example, screenshot, or link to a repository:
```
info Installing "/Users/emmet/Library/Developer/Xcode/DerivedData/pazchurch-dfffuyvjrmhokibjznhcqhpichmy/Build/Products/Debug-iphonesimulator/pazchurch.app"
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=2):
Failed to install the requested application
An application bundle was not found at the provided path.
Provide a valid path to the desired application bundle.
Print: Entry, ":CFBundleIdentifier", Does Not Exist
error Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier /Users/emmet/Library/Developer/Xcode/DerivedData/pazchurch-dfffuyvjrmhokibjznhcqhpichmy/Build/Products/Debug-iphonesimulator/pazchurch.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist
. Run CLI with --verbose flag for more details.
Error: Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier /Users/emmet/Library/Developer/Xcode/DerivedData/pazchurch-dfffuyvjrmhokibjznhcqhpichmy/Build/Products/Debug-iphonesimulator/pazchurch.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist

    at checkExecSyncError (child_process.js:629:11)
    at Object.execFileSync (child_process.js:647:13)
    at runOnSimulator (/Users/emmet/projects/github/pazchurch/node_modules/@react-native-community/cli-platform-ios/build/commands/runIOS/index.js:191:45)
    at process._tickCallback (internal/process/next_tick.js:68:7)
error Command failed with exit code 1.
```

**Comment by safaiyeh** [comment]
Hi @oeduardoal I noticed you submitted a PR in the cli repo to address this error. Would that resolve this issue as well?

**Comment by oeduardoal** [comment]
> Hi @oeduardoal I noticed you submitted a PR in the cli repo to address this error. Would that resolve this issue as well?

Yes, in reality, it is an error that occurs in the cli!

**Comment by safaiyeh** [comment]
Great! I'll close this issue. I appreciate your work on it in the cli.

**Comment by oeduardoal** [comment]
> Great! I'll close this issue. I appreciate your work on it in the cli.

Ok!

**Comment by ilyagru** [comment]
Guys, how can I upgrade to get the fix? I'm using React Native 0.63.2. And there is no newer versions available?
